torch/onnx/symbolic_helper.py

```diff
@@ -215,6 +215,10 @@ def _is_tensor(x):
 def _is_tensor_list(x):
     return isinstance(x.type(), torch._C.ListType) and isinstance(x.type().getElementType(), torch._C.TensorType)
 
+def _is_scalar_list(x):
+    return isinstance(x.type(), torch._C.ListType) and \
```

Comment:
Can you add comments to explain how you're checking types here? Maybe we can make it a bit more clear?